javascript/javascript3/week3/art.html

```diff
@@ -0,0 +1,69 @@
+<html>
```

Comment:
In general it is a good idea to not mix html/css files and javascript code. For easy of reading and mentainance it's a good practice to have them separately

Reply:
omg yes